src/openrct2-ui/windows/Land.cpp

```diff
@@ -354,7 +354,10 @@ static void window_land_paint(rct_window* w, rct_drawpixelinfo* dpi)
     {
         x = w->x + previewWidget->left;
         y = w->y + previewWidget->top;
-        gfx_draw_sprite(dpi, SPR_LAND_TOOL_SIZE_0, x, y, 0);
+        if (gLandToolSize % 2 == 1)
```

Comment:
Maybe we should use `is_power_of_2(gLandToolSize)` instead. Then I'd advise swapping the statements so we don't have to negate.

Or we can do a ternary:
```c++
gfx_draw_sprite(dpi, is_power_of_2(gLandToolSize) ? SPR_G2_MOUNTAIN_TOOL_EVEN : SPR_G2_MOUNTAIN_TOOL_ODD, x, y, 0);
```

Reply:
Is power of two isn't the same 2, 4, 8,... This should be 2,4,6,8,...

Reply:
I'd prefer keeping it as is over a single one-liner. The compiler is probably smart enough to check the least significant bit and optimize both calls into one anyway.

Reply:
I wouldn't mind determining the sprite to use in a separate statement, either. But I agree with Broxzier in that I would rather not use ternary expressions inside a function call.